crates/ef-testing/src/models/case.rs

```diff
@@ -123,6 +123,18 @@ impl BlockchainTestCase {
             .and_then(|transaction| transaction.gas_price)
             .map(|gas_price| gas_price.0)
             .unwrap_or_default();
+        let max_priority_fee_per_gas = maybe_transaction
+            .and_then(|transaction| transaction.max_priority_fee_per_gas)
+            .map(|max_priority_fee_per_gas| max_priority_fee_per_gas.0)
+            .unwrap_or_default();
+        let max_fee_per_gas = maybe_transaction
+            .and_then(|transaction| transaction.max_fee_per_gas)
+            .map(|max_fee_per_gas| max_fee_per_gas.0)
+            .unwrap_or_default();
+        // From EIP-1559: priority fee is capped because the base fee is filled first
```

Comment:
maybe add the link https://eips.ethereum.org/EIPS/eip-1559

and tbh I don't understand your docstring!

> priority fee is capped by the user's `max_fee_per_gas` parameter

?

Reply:
I will add the link, the comment comes from the EIP itself.